setup.sh

```diff
@@ -85,7 +85,11 @@ then
   # this gives you voms-proxy-*, xrdcp and other grid tools
   source /cvmfs/grid.cern.ch/etc/profile.d/setup-cvmfs-ui.sh
   # ROOT 6, voms-proxy-init and other things
+<<<<<<< HEAD:bin/env.sh
+  source /cvmfs/sft.cern.ch/lcg/views/LCG_94/x86_64-slc6-gcc62-opt/setup.sh
+=======
   source /cvmfs/sft.cern.ch/lcg/views/${LCG_VERSION}/${LCG_ARCH}/setup.sh
```

Comment:
this should be the correct one

Reply:
Oh whoops, sorry, I missed this conflict.

Reply:
This should be solved now.